### ISSUE TITLE
Affichage de TOUTES les rétroactions et non la rétroaction spécifique au choix de réponse

### ISSUE THREAD

**Issue by fuhrmanator** [ets-cfuhrman-pfe/EvalueTonSavoir#85]
Ce n'est pas claire ce qui est désiré (probablement juste une seule rétroaction pour le choix). Voici l'image saisie :

![image](https://github.com/user-attachments/assets/3108c706-089c-4978-996f-277c0ef8da0a)

**Comment by GenevieveSavard** [comment]
Oui, j'aimerais que seule la rétroaction spécifique au choix de réponse coché soit affichée, en plus de la rétroaction générale (précédée de 4# en GIFT).
